### ISSUE TITLE
[FEATURE] Offline Disk Integration

### ISSUE THREAD

**Issue by kassah** [spacedriveapp/spacedrive#63]
**Is your feature request related to a problem? Please describe.**
A lot of photographers and other media professionals keep copies of videos and photos in offline hard disks. Enabled by hard drive docks that allow easy swapping.

**Describe the solution you'd like**
It would be really nice if spacedrive could also manage indexing, redundancy and previews of offline drives. When you go for a file that was stored on external drive in your collection, it would ask for one of the drives the file exists on to be plugged in, listing the possibilities. A major part that I desire is that it not matter which computer you're on as long as you have access to the drive itself. Bonus points for keeping track of the location of drives, and allowing for redundancy rules around separate storage location of the drives (set of drives at home, set of drives at office or friends home/safety deposit box).

**Describe alternatives you've considered**
Labeling hard drives, and trying to find the right one. Manually updated spreadsheet.

**Additional context**
This would allow for photo/video libraries that don't have to always be online or in cloud. Allowing efficient, cost effective archival storage, especially for large files.

**Comment by jamiepine** [comment]
Absolutely, this has always been one of my key motivations for Spacedrive, to search offline hard drives. This already works in the app today, not at gracefully as it will eventually. But, you can add a location for an external drive, it will scan it. once you unplug the drive the directory structure, file metadata and thumbnail images will still exist in the app.
